homeassistant/components/songpal/__init__.py

```diff
@@ -1 +1,25 @@
 """The songpal component."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import HomeAssistantType
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config) -> bool:
```

Comment:
Once you move the config to a top level item, this function would call the `async_init` function with the source being `SOURCE_IMPORT`. Refer to the `vizio` links I set you for an idea on how this would work